### PR TITLE
Amend to have multiple sections

### PR DESCRIPTION
**Pull request by marksie1988**
Changed to allow multiple sections  within the json language files.

### CODE REVIEW

**Comment by iantrich** [comment]
@marksie1988 need the build error fixed before this can be merged

**Comment by marksie1988** [comment]
Im not sure why the build errors are showing, I only changed the localize file which has no issues, im using it without build errors in my custom card

File: src/localize/localize.ts [hunk 1]
```diff
@@ -7,20 +7,18 @@ var languages = {
 };
 
 export function localize(string: string, search: string = '', replace: string = '') {
-  const section = string.split('.')[0];
-  const key = string.split('.')[1];
 
   const lang = (localStorage.getItem('selectedLanguage') || 'en').replace(/['"]+/g, '').replace('-', '_');
 
   var tranlated: string;
```
Comment (bramkragten) [inline src/localize/localize.ts:13]:
```suggestion
  var translated: string;
```

Comment (bramkragten) [reply src/localize/localize.ts]: You should update the other uses of this variable too.

**Comment by marksie1988** [comment]
Ok i found what was different from the code in my working card, I have now updated it and the checks are passing.

**Pull request merged**